client/landing/stepper/declarative-flow/site-migration-flow.ts

```diff
@@ -235,8 +237,15 @@ const siteMigration: Flow = {
 		}
 
 		const goBack = () => {
-			navigate( STEPS.SITE_MIGRATION_IDENTIFY.slug );
-			// NA
+			switch ( currentStep ) {
+				case STEPS.SITE_MIGRATION_IDENTIFY.slug: {
```

Comment:
You can test the exit from as similar this case https://github.com/Automattic/wp-calypso/blob/a674a1e2c2aaf77001ee191addba7bf10509d4c7/client/landing/stepper/declarative-flow/test/site-migration-flow.tsx#L48

Reply:
Thanks for the example! That's updated now.